src/main/resources/scipooldev.properties

```diff
@@ -1,5 +1,7 @@
 SYNAPSE_OAUTH_CLIENT_ID=100055
-TEAM_TO_ROLE_ARN_MAP=[{"teamId":"273957","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}]
+TEAM_TO_ROLE_ARN_MAP=[{"teamId":"3407239","roleArn":"arn:aws:iam::465877038949:role/ServiceCatalogEndusers"}, #scipool-admin
```

Comment:
that's not going to work.  the comment is still inside of the json.  you need to put the comment outside, like this...

```
SYNAPSE_OAUTH_CLIENT_ID=100055
# 3407239 = scipool-admin 
# 3409012 = scipooldev-internal 
# 3409013 = scipooldev-external 
TEAM_TO_ROLE_ARN_MAP=[...]
AWS_REGION=us-east-1
```

Reply:
Sorry... should have read the doc you linked.